node/src/components/block_synchronizer/block_acquisition_action.rs

```diff
@@ -296,7 +300,469 @@ impl BlockAcquisitionAction {
                 validator_weights,
                 signatures,
                 is_historical,
+                legacy_required_finality,
             ),
         }
     }
 }
+
+fn enough_signatures(
+    signature_weight: SignatureWeight,
+    legacy_required_finality: LegacyRequiredFinality,
+    is_checkable: bool,
+    is_historical: bool,
+) -> bool {
+    use SignatureWeight::*;
```

Comment:
We don't tend to do this (except in the case of tests where we generally do `use super::*;`)

Reply:
Done